t/full-stack.t

```diff
@@ -71,41 +65,32 @@ plan skip_all => 'set TEST_PG to e.g. DBI:Pg:dbname=test" to enable this test' u
 my $workerpid;
 my $wspid;
 my $livehandlerpid;
-my $sharedir = setup_share_dir($ENV{OPENQA_BASEDIR});
-
 sub turn_down_stack {
-    for my $pid ($workerpid, $wspid, $livehandlerpid) {
-        next unless $pid;
-        kill TERM => $pid;
-        waitpid($pid, 0);
-    }
+    kill_service($_) for ($workerpid, $wspid, $livehandlerpid);
```

Comment:
I know my PR is not in a good shape but I hade something similar done already in #2768 . Should we follow with my changes or yours? I will try to extract the relevant part from #2768

Reply:
I prefer my change because it doesn't introduce a new function and just reuses `kill_service`.

Reply:
Fine. I can also rebase on that. Let's see who is faster ;)